### ISSUE TITLE
The core.release pipeline cannot write any tag back

### ISSUE THREAD

**Issue by Piiit** [idm-suedtirol/bdp-core#59]
See https://jenkins.testingmachine.eu/job/it.bz.opendatahub.bigdataplatform/job/core.release/5/console

```
+ git commit -m Version 2.0.0

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <(null)>) not allowed
```

**Comment by bertolla** [comment]
Injecting the jenkin user and using it's credentials might be the solution, since it's really jenkins which does the work

**Comment by alexlanz** [comment]
I did that already, but I'm not sure what went wrong. I will take a look at it next week.

**Comment by alexlanz** [comment]
Should be fixed with #62! I will check it on the next release.
